lib/dry_twitter/views/sign_in.rb

```diff
@@ -0,0 +1,22 @@
+require "dry_twitter/view/controller"
+
+module DryTwitter
+  module Views
+    class SignIn < DryTwitter::View::Controller
+      configure do |config|
+        config.template = "sign_in"
+      end
+    end
+
+    def locals(options = {})
+      if options.size > 0
+        super.merge(
+            user: options[:input]["user"],
+            errors: options[:input][:errors]
+        )
+      else
+        super.merge(user: nil, errors: nil)
```

Comment:
We can use a null guard operator to make this nicer.

Reply:
TIP: look at the [dig](https://ruby-doc.org/core-2.5.0/Hash.html#method-i-dig) method.